index.html

```diff
@@ -164,6 +164,30 @@ <h3>Skills</h3>
                     <div class="bulb"></div>
                 </div>
             </div>
+        </section>
+    <section class="section-4">
+        <div class="mobile-form-wrapper">
+            <h1 class="get-started">Get Started</h1>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloribus architecto, atque laudantium autem vel explicabo soluta quos fugit quod distinctio. Ea ullam ut facere numquam minima dignissimos? Mollitia, esse iure.</p>
```

Comment:
Please make sure you have padding on the right side of the paragraph. Check the differences below.
<img width="931" alt="Screenshot 2023-01-19 at 8 34 11 PM" src="https://user-images.githubusercontent.com/25887253/213519008-e0141741-cca6-4ba2-b231-cce774efe695.png">
<img width="322" alt="Screenshot 2023-01-19 at 8 34 55 PM" src="https://user-images.githubusercontent.com/25887253/213519015-bc721c5b-7655-469f-b26c-95b6e7f96c59.png">

Good job with the font-family but let's fix the font-size to match the Figma template.
<img width="1010" alt="Screenshot 2023-01-19 at 8 36 44 PM" src="https://user-images.githubusercontent.com/25887253/213519325-41a94a30-3fdf-40f1-876d-6a0653ded1a1.png">


<img width="814" alt="Screenshot 2023-01-19 at 8 36 33 PM" src="https://user-images.githubusercontent.com/25887253/213519321-4cea8d63-2f46-4982-b4a7-cfca9914e51b.png">

Reply:
@huseyinbiyik i think its the max-width i set wrong not that it needs padding thanks for the suggestion though

Reply:
@huseyinbiyik please can you check again, i dont think the font size is different from the figma file, please confirm this

Reply:
Hey @stino-x ,
They are different ↓
<img width="472" alt="Screenshot 2023-01-19 at 10 34 22 PM" src="https://user-images.githubusercontent.com/25887253/213542757-ca24409c-e44b-4b53-bc54-42749c469076.png">
<img width="576" alt="Screenshot 2023-01-19 at 10 35 04 PM" src="https://user-images.githubusercontent.com/25887253/213542761-36fb3eaa-e088-430f-ae92-ad0098486089.png">

Reply:
@huseyinbiyik so just to confirm i set it to 14px just like you said but it looks very different from what is in the figma file visually, am i still supposed to change it?